src/main/osd/osd.h

```diff
@@ -229,6 +231,13 @@ typedef enum {
     OSD_DISPLAYPORT_DEVICE_MSP,
 } osdDisplayPortDevice_e;
 
+typedef enum {
+    OSD_ATTR_NORMAL = 0,
```

Comment:
Can / should we combine this with `DISPLAYPORT_ATTR_`?

Reply:
Ah, these are obsolete; have already been migrated to `DISPLAYPORT_ATTR_`.